code-scanning/properties/cloudrail.properties.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "cloudrail",
+    "creator": "Indeni Cloudrail",
+    "description": "Cloudrail, by Indeni, can be used to scan your infrastructure-as-code files for potential security and compliance issues. The Cloudrail action is often used as part of both CI workflows (on pull_request) and on CD workflows to identify potential issues. Cloudrail's output uses the SARIF format, which will surface the scan's results directly inside your pull request.",
```

```suggestion
    "description": "Cloudrail can be used to scan your infrastructure-as-code files for potential security and compliance issues. The Cloudrail action is often used as part of both CI workflows (on pull_request) and on CD workflows to identify potential issues.",
```
As you can see from the rendered tile we always show `by Indeni`, and by being in the Code Scanning setup process all workflows will be using SARIF output and surfacing results in the GitHub UI. 😄